### DISCUSSION TITLE
missing functions in helper script.

### DISCUSSION THREAD

**Discussion by gugamaas** [Guilouz/Creality-Helper-Script-Wiki#586] [Issues]
hi folks.
i follow the installation procedure and my helper script dont have all the options shown in the wiki page.

![image](https://github.com/Guilouz/Creality-Helper-Script-Wiki/assets/170686681/46d7253d-2fb7-46bf-ae30-e68f1bc57f88)

i miss KAMP options.

anyone knows what is necessary to do?

tnks

**Comment by Guilouz** [comment]
Depending printer you use, if tou use KE it’s normal

**Reply by gugamaas** [reply]
i have and SE with root nebula.
i factory reset my printer and reroot it because of a problem, but before i did de reset i was able to instal KAMP.
now was it removed?

tnks for your reply.

**Reply by Guilouz** [reply]
KAMP is only possible for K1 Series and Ender3-V3, I don't have KE or SE so can't configure it.

**Reply by fbaudens** [reply]
Hello. Is KAMP available on CR10-SE ?

**Comment by ALTC-AmLearningToCode** [comment]
I have KAMP half installed on my KE, it is fully working but I cannot get the update manager to work. It was a tough process of manually importing the KAMP files and hoping they worked, which they did. Would be great for it to be in the helper script to get it installed correctly instead of whatever i did to make it work.

**Reply by ALTC-AmLearningToCode** [reply]
I found how I did it. due to unfortunate circumstances, I had to reset my printer. I found that the ssh commands that are given on github do not work directly. Assuming you have mainsail installed, in the printer files create a folder called "KAMP", then paste these modified commands into the ssh:



cd
 
git clone https://github.com/kyleisah/Klipper-Adaptive-Meshing-Purging.git
 
ln -s ~/Klipper-Adaptive-Meshing-Purging/Configuration /usr/data/printer_data/config/KAMP

cp ~/Klipper-Adaptive-Meshing-Purging/Configuration/KAMP_Settings.cfg /usr/data/printer_data/config/KAMP_Settings.cfg



And as for the update manager, I simply changed the location for the update location.



[update_manager Klipper-Adaptive-Meshing-Purging]
type: git_repo
channel: dev
path: /root/Klipper-Adaptive-Meshing-Purging
origin: https://github.com/kyleisah/Klipper-Adaptive-Meshing-Purging.git
managed_services: klipper
primary_branch: main



Changing from ~/Klipper-...
To /root/Klipper-...
